public/components/panel/workstationPanel/gridComponents.js

```diff
@@ -13,10 +13,34 @@ const GridSampleObject = {
   sampleIsActive: true,
   sampleAudioDelay: 0,
   sampleAudioStart: 0,
-  sampleAudioLength: 0,
+  sampleAudioLength: -1,
   sampleAudioBuffer: AudioBuffer,
 }
 
+const GridBeatObject = {
```

Comment:
Wouldnt we want to declare this an interface? (or is that only useful in typescript where VSCode can autocomplete

Reply:
Yeah its a workaround to get autocomplete in javascript for vscode